Source/WebCore/dom/CharacterData.cpp

```diff
@@ -224,4 +235,15 @@ void CharacterData::dispatchModifiedEvent(const String& oldData)
     InspectorInstrumentation::characterDataModified(protectedDocument(), *this);
 }
 
+bool CharacterData::containsOnlyASCIIWhitespace() const
+{
+    if (hasStateFlag(StateFlag::ContainsOnlyASCIIWhitespaceKnown))
+        return hasStateFlag(StateFlag::ContainsOnlyASCIIWhitespace);
+
+    bool hasOnlyWhitespace = m_data.containsOnly<isASCIIWhitespace>();
+    const_cast<CharacterData*>(this)->setStateFlag(StateFlag::ContainsOnlyASCIIWhitespace, hasOnlyWhitespace);
```

Comment:
`setStateFlag` has `const` qualifier so we shouldn't have to do have this const_cast.